src/components/PivotTable/PivotTableValueCell.js

```diff
@@ -46,7 +49,8 @@ export const PivotTableValueCell = ({
     // TODO: Add support for 'INTEGER' type (requires server changes)
```

Comment:
Does this comment still applies?
`isNumericValueType` returns `true` for integer types.